udrzba/_pomocne.php

```diff
@@ -50,7 +74,7 @@ function nasad(array $nastaveni) {
 
   // nahrání souborů
   msg('synchronizuji soubory na vzdáleném ftp');
-  $souborNastaveniDeploymentu = 'tmp-' . mt_rand();
+  $souborNastaveniDeploymentu = '_tmp-' . mt_rand();
```

Comment:
Tím že to má teď podtržítko na začátku, tak splňuje tu úvodní pomínku pro ignore `/_*` a tím se tohle nastavení nekopíruje na server.